src/kademlia/get_item.cpp

```diff
@@ -78,7 +88,7 @@ void get_item::got_data(bdecode_node const& v,
 		// and it's immutable
 
 		m_data.assign(v);
-		bool put_requested = m_data_callback(m_data);
+		bool put_requested = m_data_callback(m_data, false);
```

Comment:
but for immutable data, every response is authoritative.. so, I understand this path now. m_data hasn't been set yet, which means this is the first response to an immutable lookup. I would expect authoritative to always be true here (since there is no other value the lookup can return).